cmd/ferretdb/main.go

```diff
@@ -48,10 +48,11 @@ import (
 //
 // Keep order in sync with documentation.
 var cli struct {
-	Version  bool   `default:"false" help:"Print version to stdout and exit." env:"-"`
-	Handler  string `default:"pg" help:"${help_handler}"`
-	Mode     string `default:"${default_mode}" help:"${help_mode}" enum:"${enum_mode}"`
-	StateDir string `default:"."               help:"Process state directory."`
+	Version            bool   `default:"false"           help:"Print version to stdout and exit." env:"-"`
+	Mode               string `default:"${default_mode}" help:"${help_mode}"                      enum:"${enum_mode}"`
+	Handler            string `default:"pg"              help:"${help_handler}"`
+	StateDir           string `default:"."               help:"Process state directory."`
+	AppendCoverageMeta bool   `default:"false"           help:"Append test coverage"`
```

Comment:
Let's remove that flag and always collect coverage in the current directory when built with `ferretdb_debug` build tag

Reply:
I am not sure how to do this. How can I enable cover mode in a debug mode or How can I write coverage data to a directory only in the debug build?

Reply:
> How can I enable cover mode in a debug mode or How can I write coverage data to a directory only in the debug build?

I think both things should be enabled by different Task targets. We still not need our own flag for that.